ci/citest/acceptance-test/multibox/10.0.100.17-vdc-executor-qemu-ovs/build.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+
+export ENV_ROOTDIR="$(cd "$(dirname $(readlink -f "$0"))/.." && pwd -P)"
+export NODE_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
+TMP_ROOT="${NODE_DIR}/tmp_root"
+
+. "${ENV_ROOTDIR}/config.source"
+. "${NODE_DIR}/vmspec.conf"
+. "${ENV_ROOTDIR}/ind-steps/common.source"
+
+IND_STEPS=(
+    "box"
+    "ssh"
+    "hosts"
+    "disable-firewalld"
+    "epel"
+    "openvswitch"
+    "qemu"
+    "kvm"
```

Comment:
It was because I mistook nested functionality with kvm module getting loaded, theoretically qemu executor does not require kvm but I can merge these if that makes more sense,